src/EnergyPlus/Coils/CoilCoolingDXCurveFitPerformance.cc

```diff
@@ -45,15 +45,19 @@
 // OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-#include <ObjexxFCL/gio.hh>
+#include <EnergyPlus/Coils/CoilCoolingDX.hh>
```

Comment:
This is a circular reference.  I am glad it is compiling, but we should not be doing this.  The child components don't need to get access to the parents.  If this is purely to get access to the coil operating mode integers, those should be moved to a separate class where all of the coil classes can access it.  Or those integers should be pushed lower, maybe into this file.